test/tests/trackers.js

```diff
@@ -19,17 +19,22 @@
     // invalid URL
     { url: 'sdfsdfs', block: false}
   ];
-  
+
   QUnit.test("block url", function (assert) {
       // turn social blocking on for this test
       bkg.settings.updateSetting('socialBlockingIsEnabled', true);
-      
-      basicBlocking.forEach(function(test) {
-          bkg.settings.updateSetting('trackerBlockingEnabled', true);
-          var toBlock = bkg.trackers.isTracker(test.url, fakeTab, fakeRequest);
-          toBlock = toBlock ? true : false;
-          assert.ok(toBlock === test.block, 'url should be blocked');
-      });
+      var done = assert.async()
+
+      setTimeout(function () {
+          basicBlocking.forEach(function(test) {
+              bkg.settings.updateSetting('trackerBlockingEnabled', true);
+              var toBlock = bkg.trackers.isTracker(test.url, fakeTab, fakeRequest);
+              toBlock = toBlock ? true : false;
+              assert.ok(toBlock === test.block, 'url should be blocked');
+
+              done()
+          });
+      }, 1000);
```

Comment:
This was erroring out since the lists haven't loaded by the time this was hit.

This fix is brittle. I've seen it fail a couple of times, but given I'm gonna update the tests soon, I think it's good enough.